xmipptomo/protocols/protocol_score_coordinates.py

```diff
@@ -113,6 +121,7 @@ def detectOutliers(self):
             self.scoreOutliers[idn][1] = z_scores[idn]
 
     def detectCarbonCloseness(self, coordinates):
+        pwutils.makePath(self._getExtraPath('carbonMask'))
         self.scoreCarbon = []
         for tomoName in self.tomoNames:
```

Comment:
The class has not _self.tomoNames_ definition

Reply:
Hi @albertmena,

The attribute `self.tomoNames` is initialized in the method `computeParams` in line 101, which is called as a step inside the `_insertAllSteps` method (line 94). This step is called before `detectCarbonCloseness`, are you getting any error related to it? In my local installation, the test runs without issues. Thanks!

Reply:
looks like some local issue, the test works on galileo_shadow